app/scripts/cart/templates/add-to-cart-all-button.html

```diff
@@ -2,7 +2,7 @@
   id="add-to-cart-button"
   class="btn btn-primary"
   data-ng-click="ctrl.addAll(ctrl.files)"
-  data-ng-disabled="!ctrl.hasFiles"
+  data-ng-disabled="ctrl.size === '0' || (ctrl.files && ctrl.files.length === 0)"
```

Comment:
https://github.com/NCI-GDC/portal-ui/pull/1006/commits/1571c3f4e0805bb518dec2edc10ae908af129305#diff-a4de33ae03db0ff7e45bd7fc76f505e1L66 was converting size to a string, updated